prow/caching.md

```diff
@@ -0,0 +1,50 @@
+# Caching dependencies in Prow
+
+Caching dependencies in prow works only when the [Pod Utilities][0] are enabled. A
+valid GCS Bucket has to be configured via `gcs_configuration` either in the Prow
+jobs `decoration_config` or in [Planks][1] `default_decoration_configs`.
+
+[0]: /prow/pod-utilities.md
+[1]: /prow/cmd/plank/README.md
+
+Saving and restoring the cache will be done in the `clonerefs` and `sidecar` pod
+utility containers, where:
+
+- `clonerefs` tries to restore the cache after the clone has been done. Multiple
```

Comment:
Please clarify what happens if there is no cache and what happens if there was an error accessing the cache

Reply:
Done :+1: